applications/asset_tracker_v2/src/cloud/cloud_codec/lwm2m/lwm2m_codec.c

```diff
@@ -85,30 +85,22 @@ int cloud_codec_init(struct cloud_data_cfg *cfg, cloud_codec_evt_handler_t event
 	return 0;
 }
 
-int cloud_codec_encode_neighbor_cells(struct cloud_codec_data *output,
-				      struct cloud_data_neighbor_cells *neighbor_cells)
+int cloud_codec_encode_cloud_location(struct cloud_codec_data *output,
+				 struct cloud_data_cloud_location *cloud_location)
 {
 	ARG_UNUSED(output);
 
 	int err;
 
-	err = lwm2m_codec_helpers_set_neighbor_cell_data(neighbor_cells);
+	err = lwm2m_codec_helpers_set_neighbor_cell_data(&cloud_location->neighbor_cells);
```

Comment:
I though LwM2M utils did support wifi access point based location? Or do you want to add it in a follow up PR perhaps? Maybe I'm mistaken. cc @SeppoTakalo

Reply:
LwM2M utils do support them but integration of WiFi+LwM2M into ATv2 hasn't been done and that's out of scope of this PR.
Just like we don't have WiFi+Azure or WiFi+AWS combinations done in ATv2. These are separate exercises from this PR.